generators/app/templates/docker-compose.yml

```diff
@@ -1,9 +1,12 @@
-<%= imageName %>:
-  dockerfile: Dockerfile.<%= environment %>
-  build: .<% if (isWebProject && portNumber) { %>
-  ports:
-    - "<%= portNumber %>:<%= portNumber %>"<% } %><% if (environment === 'debug' && volumeMap !== null) { %>
-  volumes:
-    - <%= volumeMap %><% } %>
-  labels:
-    com.<%= imageName %>.environment: "<%= environment %>"
+version: '2'
+
+services:
+  <%= serviceName %>:<% if (environment === 'debug') { %>
+    image: <%= imageName %>:debug<% } else { %>
```

Comment:
It is case sensitive (That's why i added a ToLower in the PS1) and the files are also lower case and case sensitive in the Mac scenario so i am trying to stay lowercase consistently. 

---
In reply to: [66846684](https://github.com/Microsoft/generator-docker/pull/63#discussion_r66846684) [](ancestors = 66846684)